knotx-example/src/main/resources/content/local/simple.html

```diff
@@ -0,0 +1,56 @@
+<!--
```

Comment:
This file will be not used. Local sample repository fetch simple.html from classpath. This should be example part but good solution is still not implemented.